programming-learning-tool/server/data-source.ts

```diff
@@ -0,0 +1,16 @@
+import "reflect-metadata"
+import  { DataSource }from "typeorm"
+import { ProgrammingTask } from "./models/ProgrammingTask"
+export const AppDataSource = new DataSource({
+    type: "postgres",
+    host: "localhost",
+    port: 5431,
+    username: "user",
+    password: "password",
```

Comment:
?

Reply:
det er bare en test container, dog er der et argument for at det skulle se anderledes ud på en production database. F.eks. brug af en env fil